docs/usage.md

```diff
@@ -316,6 +316,23 @@ For example, this will build all targets the same way that Xcode does:
 bazel run //:xcodeproj -- --generator_output_groups=all_targets build
 ```
 
+### `--download_intermediates`
```

Comment:
I'm not set on this name, if something else sounds better.

Reply:
Seems like a reasonable name to me